chromadb/test/property/test_embeddings.py

```diff
@@ -194,6 +247,12 @@ def upsert_embeddings(self, record_set: strategies.RecordSet) -> None:
 
         self.collection.upsert(**record_set)
         self._upsert_embeddings(record_set)
+        if not NOT_CLUSTER_ONLY:
```

Comment:
nit: if NOT_CLUSTER_ONLY and should_compact can be one line

Reply:
Changed the approach so this comment is moot now